elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -288,6 +292,12 @@ public boolean matchesId(String checkId) {
             return Collections.emptySet();
         }
 
+        //filterExpressionCheck
+        FilterExpression permissionFilter = getPermissionFilterExpression(loadClass, requestScope);
```

Comment:
I dont' think this code actually changes the load.  I think this change can be removed.  Look at how loadObjects is implemented in HibernateTransaction.